### PR TITLE
feat: update boarding pass info to show airport checkin option

### PR DESCRIPTION
**Pull request by Radek-Palisa**
#689 

About the tooltip:
I have imported the tooltip component from FE with its original styles. Adjustments to its css are done there where the component is imported, so in this case BoardingPassesDescription.js includes some css overwrites/additions to the tooltip.<br/><br/><br/><url>LiveURL: https://smartfaq-update-boarding-passes-info.surge.sh</url>

### CODE REVIEW

File: src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/BoardingPassesDescription.js [hunk 1]
```diff
@@ -30,6 +36,49 @@ const BoardingPassesDescription = ({ data, mmbUrl }: Props) => {
...
+            <Download size="medium" customColor="#00a991" />
+            Download
+          </a>
+        );
+      case 'IN_FUTURE':
+        return <p>Available {availableAt}</p>;
+      case 'AT_AIRPORT':
+        return (
+          <HoverHelpTooltip
+            tooltip="You will have to check-in for free at the aiport"
```
Comment (jalbertsr) [inline src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/BoardingPassesDescription.js]: small typo `aiport` -> `airport`

**Comment by jalbertsr** [comment]
LGTM, fix typo and rebase 👍

**Comment by Radek-Palisa** [comment]
@jalbertsr typo fixed

**Pull request merged**